components/camera/replaypcd/replaypcd.go

```diff
@@ -185,9 +281,14 @@ func (replay *pcdCamera) Stream(ctx context.Context, errHandlers ...gostream.Err
 	return stream, errors.New("Stream is unimplemented")
 }
 
-// Close stops replay camera and closes its connections to the cloud.
+// Close stops replay camera, closes the channels and its connections to the cloud.
 func (replay *pcdCamera) Close(ctx context.Context) error {
+	// Close cache channel
+	close(replay.cacheCh)
```

Comment:
this Close should prevent itself from being closed again

Reply:
Outdated as channel is being removed